Sources/docc/DocCDocumentation.docc/Commands/convert.md

```diff
@@ -0,0 +1,175 @@
+# ``convert``
+
+Convert documentation markup, assets, and symbol information into a documentation archive.
+
+@Metadata {
+    @PageImage(purpose: icon, source: command-icon)
+}
+
+## Arguments, Flags, and Options
+
+### Inputs & Outputs
+
+- term `<catalog-path>`:
+  Path to the documentation catalog ('.docc') directory.
+
+- term `--additional-symbol-graph-dir <additional-symbol-graph-dir>`:
+  A path to a directory of additional symbol graph files.
+
+- term `-o, --output-path, --output-dir <output-path>`:
+  The location where the documentation compiler writes the built documentation.
+
+### Availability Options
+
+- term `--platform <platform>`:   
+  Specify information about the current release of a platform.
+  
+  Each platform's information is specified via separate `--platform` values using the following format: `name={platform name},version={semantic version}`.
+
+  Optionally, the platform information can include a `beta={true|false}` component. If no beta information is provided, the platform is considered not in beta.
+
+### Source Repository Options (Swift-DocC 5.8+)
+
+- term `--checkout-path <checkout-path>`:
+  The root path on disk of the repository's checkout.
```

Comment:
How does using this option affect what `convert` does?

I also have the same question for the next two options too 😄